README.md

```diff
@@ -96,9 +96,9 @@ Ensure the following elements are present in your `manifest.yml`. The `run_migra
 To seed fund & round data to db
 
 ```
-docker exec -ti <fund_store_container_id> python -m scripts.load_cof_r2
-docker exec -ti <fund_store_container_id> python -m scripts.load_cof_r3w1
-docker exec -ti <fund_store_container_id> python -m scripts.load_ns_r2
+docker exec -ti <fund_store_container_id> python -m scripts.fund_round_loaders.load_cof_r2
```

Comment:
We could do something like this to grab the container id automatically for the assessment store?   

docker exec -ti **$(docker ps -qf "name=fund-store")** python -m scripts.fund_round_loaders.load_cof_r2

Reply:
i like your thinking @ramsharma-prog ill add this 👍

Reply:
Please be aware if multiple containers has "fund-store" like names, then the command wont work